### DISCUSSION TITLE
Change skip for./ rev. interval seconds

### DISCUSSION THREAD

**Discussion by byte-lingual** [sampotts/plyr#2632] [Ideas]
Plyr and users will benefit greatly from being able to change the skip forward/ reverse interval to a more user specific situation. Often times the 10 seconds is too far forward or back in a video or clip. Examples can be watching a ufc fight and a user wants to see a move that was just performed. The viewer is watching to learn how the move was performed or just to see that highlighted replay. Another example is a student learning the pronunciation of a foreign language vocab. Maybe the viewer wishes to see how something is being made and a few seconds back allows that quick repetition detailed learning. I recommend intervals of 1, 2, 3, 5, and 10. 1 to 2 seconds is great for learning a foreign language. 3 to 5 seconds is great for learning or catching a highlighted moment in a game. Thanks for having the same vision for a superb media player viewers have been waiting for. It's coming along really good.

**Comment by belbernard** [comment]
I haven't even been able to display skip buttons allowing for jumps by 10 seconds.
There is a "seekTime" option that (according to ChatGPT) is meant to modify these values. Unfortunately I can't see skip buttons. Do you have code showing how to display them?
